crates/katana-rpc/src/katana/api.rs

```diff
@@ -16,4 +17,7 @@ impl From<KatanaApiError> for Error {
 pub trait KatanaApi {
     #[method(name = "generateBlock")]
     async fn generate_block(&self) -> Result<(), Error>;
+
+    #[method(name = "predeployed_accounts")]
```

Comment:
```suggestion
    #[method(name = "predeployedAccounts")]
```

Reply:
I have choose snake case like [devnet](https://0xspaceshard.github.io/starknet-devnet/docs/guide/accounts/#fetch-predeployed-accounts)
```
GET /predeployed_accounts
```

But for sure I can change

Reply:
devnet is a rest endpoint though. for jsonrpc we should follow the convention of the other endpoints

Reply:
PR updated & rebased on main